### PR TITLE
Removed manipulation of the `value` constant passed by the CMS

### PR DESCRIPTION
**Pull request by jimmynotjim**
- Fix to help #1393 along.
- There may be other areas we need to replace value where we were improperly manipulating it.

### CODE REVIEW

File: cfgov/jinja2/v1/_includes/molecules/global-header-cta.html [hunk 1]
```diff
@@ -8,11 +8,15 @@
 
    Creates markup for Global Header Call to Action molecule.
 
-   value.is_in_list: Whether the molecule appears in a list or not.
+   horizontal: Whether the molecule appears horizontally or vertically.
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/_includes/molecules/global-header-cta.html]: Update `is_horizontal` docs

File: cfgov/jinja2/v1/_includes/templates/nav/primary-nav.html [hunk 2]
```diff
@@ -45,15 +37,7 @@
...
-            'global_eyebrow': {
-                'is_in_list': true
-            }
-
-        })
-    %}
-    {% set value = global_dict.global_eyebrow %}
-    {% include 'molecules/global-eyebrow.html' with context %}
+    {% from 'molecules/global-eyebrow.html' import render as eyebrow with context %}
+    {{ eyebrow() }}
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/_includes/templates/nav/primary-nav.html]: To following existing conventions this should be `global_eyebrow.render()` so the function call name isn't dependent on the context in which it is imported.

Comment (jimmynotjim) [inline cfgov/jinja2/v1/_includes/templates/nav/primary-nav.html]: I was following @kurtw's example in https://github.com/cfpb/cfgov-refresh/blob/flapjack/cfgov/jinja2/v1/browse-filterable/index.html#L29. I never know which is the right way. Kurt, can you fill us in?

Comment (kurtw) [inline cfgov/jinja2/v1/_includes/templates/nav/primary-nav.html]: I think Ans' approach is more readable and easier to remember.

Comment (jimmynotjim) [inline cfgov/jinja2/v1/_includes/templates/nav/primary-nav.html]: Thanks, done

**Comment by anselmbradford** [comment]
:+1:

**Comment by sebworks** [comment]
:+1:

**Pull request merged**